src/lib/language.ml

```diff
@@ -279,7 +303,7 @@ let rec to_shell: type a. _ -> a t -> string =
           [
             "while :;"; " do case $1 in";
             "-h|-help|--help) ";
-            sprintf "export %s_help=0 ; " prefix;
+            sprintf "export %s_help=true ; " prefix;
```

Comment:
How do you feel about replacing `sprintf` with `s` inside this module to help readability?

I found it a bit easier, but it might not work for you.